vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt

```diff
@@ -70,47 +71,24 @@ class HomeActivityViewModel @AssistedInject constructor(
     companion object : MavericksViewModelFactory<HomeActivityViewModel, HomeActivityViewState> by hiltMavericksViewModelFactory()
 
     private var checkBootstrap = false
-    private var onceTrusted = false
 
-    init {
-        cleanupFiles()
-        observeInitialSync()
-        checkSessionPushIsOn()
-        observeCrossSigningReset()
+    override fun handle(action: HomeActivityViewActions) {
```

Comment:
The diff is a bit strange here, at least from GitHub POW.
The method `handle` was already existing.
Also forks may suffer from this change, so it's probably better to ensure that they are minimal.

Reply:
the diff looks really painful but all it's doing is moving the handle function to the top of the file and inlining the init functions into the `ViewStarted` action

I'll revert and leave the functions above the `handle` to improve the diff

Reply:
updated including the started guard https://github.com/vector-im/element-android/pull/4370/commits/e3543131859f1365b6c24eb1f1434fdc07cb5819